CRM/Contribute/Form/AdditionalPayment.php

```diff
@@ -185,7 +185,7 @@ public function setDefaultValues() {
     }
 
     if ($this->_refund) {
-      $defaults['total_amount'] = abs($this->_refund);
+      $defaults['total_amount'] = number_format(abs($this->_refund), 2);
     }
```

Comment:
@eileenmcnaughton See the first comment :-) that you commented on.  number_format($var,2) will round to 2decimal places - so will CRM_Utils_Money::format.  I do agree that CRM_Utils_Money::format is a bit of a messy function, but that should probably be cleaned up as a separate exercise.  All money amounts that are formatted for display should really go through that function so we have one place for formatting.  Note that I can't use it when I want to write more than 2dp to the database eg. (https://github.com/civicrm/civicrm-core/pull/10641/files#diff-9bb0aa4010a43713521df361cca72fe6R841).  Though it might make sense in that link to replace my filter_var with another CRM_Utils_money function!

In this case, as the value is ONLY destined for display, I think CRM_Utils_Money::format(...onlyNumber=TRUE...) is the right function to use.